corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/api.js

```diff
@@ -5,19 +5,39 @@
  */
 
 FormplayerFrontend.module("Apps", function (Apps, FormplayerFrontend, Backbone) {
-
     var appsPromiseByRestoreAs = {};
     var appsByRestoreAs = {};
+    var predefinedAppsPromise;
+
+    function fetchAllApps(restoreAs) {
+        var appsPromise = appsPromiseByRestoreAs[restoreAs];
+        if (!appsPromise || appsPromise.state() === 'rejected') {
+            appsPromise = appsPromiseByRestoreAs[restoreAs] = $.getJSON('?option=apps');
+        }
+        return appsPromise;
+    }
+
+    function fetchPredefinedApps() {
+        /*
+          in singleAppMode we want to avoid calling the server when we switch users because
+          1. it is unnecessary, since there's only one app regardless of restoreAs user
+          2. the backend ?options=apps endpoint is not defined for single-app pages
```

Comment:
nit: spelling  `?option=apps`. Or is the previous reference incorrect (line 15)?